tools/nl/embeddings/input/undata_ilo/sheets_svs.csv

```diff
@@ -1,86 +1,33 @@
 dcid,sentence
-dc/topic/ILOCPINCYRRT,"Consumer Price Index CPI, Percentage Change From Previous Year"
-dc/topic/ILOCPINWGTRT,"National Consumer Price Index CPI, Country Weights"
-dc/topic/ILOCPIXCPIRT,National Consumer Price Index CPI
-dc/topic/ILOEAP2EAPNB,Labour Force ILO Modelled Estimates
-dc/topic/ILOEAP2WAPRT,Labour Force Participation Rate ILO Modelled Estimates
-dc/topic/ILOEAP3EAPNB,Youth Labour Force
-dc/topic/ILOEAP3WAPRT,Youth Labour Force Participation Rate
-dc/topic/ILOEAR4MMNNB,Monthly Minimum Wage
-dc/topic/ILOEARXTLPRT,Low Pay Rate
-dc/topic/ILOEES3EESNB,Youth Employees
-dc/topic/ILOEESXTMPRT,Share of Temporary Employees
-dc/topic/ILOEIP2EIPNB,Persons Outside The Labour Force ILO Modelled Estimates
-dc/topic/ILOEIP2JOBNB,"Potential Labour Force And Willing Non-jobseekers ILO Modelled Estimates, as Of Nov. 2023"
-dc/topic/ILOEIP2JOBRT,"Potential Labour Force And Willing Non-jobseekers Rate ILO Modelled Estimates, as Of Nov. 2023"
-dc/topic/ILOEIP2PLFRT,Potential Labour Force Rate ILO Modelled Estimates
-dc/topic/ILOEIP2WAPRT,"Number of Persons of Working Age Outside The Labour Force, as A Percentage of The Working-age Population Inactivity Rate, ILO Modelled Estimates"
-dc/topic/ILOEIP3EIPNB,Youth Outside The Labour Force
-dc/topic/ILOEIP3WAPRT,"Number of Young Persons of Working Age Outside The Labour Force, as A Percentage of The Working-age Population Youth Inactivity Rate"
+dc/topic/ILOEAR4MMNNB,"Monthly Minimum Wage"
+dc/topic/ILOEARXTLPRT,"Low Pay Rate"
+dc/topic/ILOEESXTMPRT,"Share of Temporary Employees"

```

Comment:
"percentage of.." if that's the intention. "share of" might not result in good match.

Reply:
Added to alternatives.